### PR TITLE
Remove deprecated egress rpc

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: livekit_egress.proto [hunk 1]
```diff
@@ -35,11 +35,6 @@ message RoomCompositeEgressRequest {
   bool audio_only = 3;          // (default false)
   bool video_only = 4;          // (default false)
   string custom_base_url = 5;   // template base url (default https://recorder.livekit.io)
-  oneof output {                // deprecated (use _output fields)
```
Comment (davidzhao) [inline livekit_egress.proto:38]: ok, we cannot deprecate this yet. this is a major change.

especially for Cloud customers, we need to ensure everyone's migrated off before pulling the trigger.

File: livekit_egress.proto [hunk 2]
```diff
@@ -267,21 +252,11 @@ message EgressInfo {
     TrackEgressRequest track = 6;
     WebEgressRequest web = 14;
   }
-  oneof result { // deprecated (use _result fields)
```
Comment (davidzhao) [inline livekit_egress.proto:270]: same for all the result fields

File: livekit_egress.proto (hunk 1 above)
Comment (frostbyte73) [reply livekit_egress.proto:38]: I can leave the fields in for now and just remove the rpc system

**Review by davidzhao — APPROVED** [review]
lgtm!

**Pull request merged**